<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormStudentMode</class>
 <widget class="QWidget" name="FormStudentMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelWelcome">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>同学欢迎你</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelStatus">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>选课</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>查询成绩和已选课程</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonReflash">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>教师姓名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditTeacherName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>课程名称：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditCourseName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>课程学分：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditCourseCredit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>分数从：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditDownGrade"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>到</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditUpGrade"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="comboBoxCourseType">
           <item>
            <property name="text">
             <string>无</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>必修</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>选修</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QPushButton" name="pushButtonQuery">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonEmpty">
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
